src/sass/app_2/_common/components/accounts_switcher.scss

```diff
@@ -113,29 +107,29 @@
         }
         &--enter, &--exit {
             opacity: 0;
-            transform: translateY(-20px);
+            transform: translateY(-2em);
         }
     }
     &__new-account {
         display: flex;
         justify-content: center;
-        margin: 8px 0 16px;
+        margin: 0.8em 0 1.6em;
 
         span {
-            padding: 8px 16px;
-            border: 1px dashed $COLOR_DARK_BLUE_2;
+            padding: 0.8em 1.6em;
+            border: 0.1em dashed $COLOR_DARK_BLUE_2;
             color: $COLOR_DARK_BLUE_2;
-            font-size: 0.75rem;
+            font-size: 1.2rem;
```

Comment:
I think in general font-size should be relative to its container, not root.

Reply:
@ashkanx thanks for the catch, this should be in `em`